### DISCUSSION TITLE
WLAN PI  can be installed in any Hardware?

### DISCUSSION THREAD

**Discussion by Mani654-s** [WLAN-Pi/feedback#79] [General Inquiries] [answered]
Can WLAN PI be installed on any hardware?
for eg. can we install this on a Lenovo AMD laptop? or a intel based laptop?

**Comment by jiribrejcha** [comment, marked as answer]
No. It only supports [this](https://userguide.wlanpi.com) arm64 hardware. It is mainly down to the driver support. We are quite specific about our Wi-Fi requirements.

Having said that, our code is open source and available on GitHub if you wish to port it to another platform.
